dashdoc-frontend/src/components/analytics/Analytics.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import * as S from '@styles';
+import Metric from '@components/analytics/Metric';
+import { formatCurrency } from '@utils/formatter';
+import { mockUserAnalytics } from '@utils/mocks/analyticsMocks';
+import { monthNames } from '@constants';
+import { useAnalytics } from '@hooks/useAnalytics';
+
+const Analytics = ({ userType }: any) => {
+  // this will be used once server is hooked up and will replace mockUserAnalytics
+  const { user_analytics } = useAnalytics();
+  return (
+    <div>
+      <S.StyledHeader fontSize='18' fontColor='black' fontWeight='bold'>
+        {monthNames[mockUserAnalytics.lastUpdated.getMonth()]} Analytics
```

Comment:
Small suggestion. This can get you the month without a months constant.
```suggestion
        {new Date().toLocaleString('default', { month: 'long' })} Analytics
```